nodejs/week3/calculator.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const app = express();
+
+app.get('/', (req, res) => {
+   res.send('<h1> Calculator </1>');
+});
+
+// ! localhost:3000/calculator/add?firstParam=1&secondParam=2
+app.get('/calculator/:params', (req, res) => {
+   const numbers = [];
+   for (const key in req.query) {
+      const number = Number(req.query[key]);
+      numbers.push(number);
+   }
```

Comment:
calculator/multiply?a=1&b=2&c=4&d=6
it works if I have different middleware like a, b, c and d ..
but not with same middleware name

Reply:
In the case where you have the same query parameter multiple times, then it will be an array.
(BTW: I think you mean query parameter, not middleware)